src/common/middlewares/jwt.middleware.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable, NestMiddleware } from '@nestjs/common';
+import { NextFunction, Request, Response } from 'express';
+import { JwtService } from '@nestjs/jwt';
+
+@Injectable()
+export class JwtMiddleware implements NestMiddleware {
+  constructor(private readonly jwtService: JwtService) {}
+
+  async use(req: Request, res: Response, next: NextFunction) {
+    const token = req.headers['authorization'];
```

Comment:
Better to follow `PascalCase` for header naming i.e
```suggestion
    const token = req.headers['Authorization'];
```